CPS-XXXX/README.md

```diff
@@ -0,0 +1,210 @@
+  This limit is a disjunction of Common Prefix and Chain Growth.
+
+  With this option, the node would select a sparse chain and would also be able to switch away from any of the youngest 2160 blocks, but only those that are younger than 36 hrs.
+
+- *EnforceChainGrowth* (EnforceCG).
+  In addition to AssumeIA, the node rejects any chain that has less than 2160 blocks per 36 hr (even if selecting it wouldn't require rolling back more than 2160 blocks).
+
+  With this option, the node would never select a sparse chain; it'd select a prefix of it with degrading density but reject the first block that is more than 36 hr younger than its 2161st ancestor.
+
+Outside of any scenario, EnforceIA and EnforceCG introduce some simplifying invariants compared to AssumeIA, simply because they are explicit.
+They also have costs, discussed in the specific scenarios below.
+
+  - The EnforceIA node can ignore chains that branch off more than 36 hr ago, even if they would only requiring rolling back at most 2160 blocks.
+
+  - The EnforceCG node can (temporarily) ignore all blocks that are more than 36 hr younger than its 2161st youngest block.
+
+It's possible that an AssumeIA node can be sufficiently resilient despite not having those additional invariants, but its security argument becomes more complicated.
+That means mistakes are more likely, and it's more difficult to correctly add features, new protocols, etc.
+
+These options should also be contrasted by considering the following scenarios.
+
+- *TwoThirdsInaccessible*.
+  Suppose more than 2/3rds of stake was knocked offline (eg by a solar flare) such that less than 1/3rd of all stake was active for more than 36 hr, but it all remained connected.
+
+    - The other scenarios all involve this one to some degree, so it serves as a baseline.
+
+    - If the adversary controls less than half of the surviving stake, then it still cannot defeat Praos during the outage.
+
+    - Note that the chain would have a growth rate less than 2160 per 36 hr but greater than zero.
+      
+    - If the whole network implemented AssumeIA, then the 2161th youngest block would be at least three times older than usual (36+ hr instead of 12 hr).
+      Since the nodes and/or other Cardano community tooling might have been designed around the guarantee of at most 36 hrs for 2160 blocks, they could fail outright or become more vulnerable to possible DoS attacks during the outage.
+
+    - If the whole network implemented EnforceIA, then the nodes would not face increased risks.
+      The 2160 disjunct in the EnforceIA definition is an optimization for when the chain is growing faster than 2160 blocks per 36 hr.
+
+    - If the whole network implemented EnforceCG, then the Recovery Plan would be unavoidable (once the affected stake was back online), since the honest nodes would eventually refuse to the newly minted blocks, since they are more than 36 hr younger than their 2161st predecessor.
```

Comment:
```suggestion
    - If the whole network implemented EnforceCG, then the Recovery Plan would be unavoidable (once the affected stake was back online), since the honest nodes would eventually refuse to select the newly minted blocks, since they are more than 36 hr younger than their 2161st predecessor.
```